packages/embark-console/src/lib/index.ts

```diff
@@ -1,14 +1,11 @@
-/*globals __*/
-import "colors";
-const env = require("../../core/env");
-const utils = require("../../utils/utils");
-const {escapeHtml, exit} = require("embark-utils");
-import { Callback } from "embark";
-import constants from "embark-core/constants.json";
-const stringify = require("json-stringify-safe");
 import { waterfall } from "async";
-import { Embark, Events } from "embark";
-import {__} from "i18n";
+import chalk from "chalk";
```

Comment:
Using `chalk` instead of a `colors` is a *really good* improvement because `colors` patches the `String` prototype and we should generally stay away from libraries that do that unless there's not another good option.

BUT it turns out there was a problem with chalk and TypeScript / Babel, until recently. See: https://github.com/chalk/chalk/pull/296#issuecomment-418789492.

The fix is to bump the typescript dependency, and I went ahead and did it for all packages in the monorepo. I also went ahead and bumped the version of TSLint.

Reply:
> Using chalk instead of a colors is a really good improvement (thanks for doing that!) because colors patches the String prototype and we should generally stay away from libraries that do that unless there's not another good option.

I thought so too. Not a huge fan of this sort of patching in JS.